ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-layering.yaml

```diff
@@ -17,9 +21,14 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
   to: machine-config-operator
+- dockerfile_literal: FROM scratch
```

Comment:
I don't think this is needed because we're importing the image from the `base_images` portion, but I could be wrong.

Reply:
I am cargo culting bits from the old mcbs branch.  I *think* the reason we need this is we want to make this image part of our "things we built" set (just like we're building a new `machine-config-operator`) to ensure that that image ends up overriding what's currently in the base release image.

Reply:
Ahh gotcha. Leave it as-is then.